docs/docs/examples/guides/04-object-detection-example.md

```diff
@@ -0,0 +1,745 @@
+      "parameter_name": "to_find",
+      "parameter_type": "string",
+      "default_value": "person"
+    },
+    {
+      "display_name": "Field of View (FoV)",
+      "description": "Camera field of view in degrees [horizontal, vertical]",
+      "parameter_name": "fov",
+      "parameter_type": "double_array",
+      "default_value": [69.0, 42.0]
+    },
+    {
+      "display_name": "Image Size",
+      "description": "Image resolution [width, height] in pixels",
+      "parameter_name": "image_size",
+      "parameter_type": "double_array",
+      "default_value": [480, 840]
+    },
+    {
+      "display_name": "Object Distance",
+      "description": "Assumed distance from the camera to the object in meters",
+      "parameter_name": "object_distance",
+      "parameter_type": "double",
+      "default_value": 0.6,
+      "dynamic": true
+    }
+  ]
+}
+```
+
+</details>
+
+Enter the component folder in terminal and run
+
+```bash 
+docker build -f aica-package.toml -t objectdetection .
+```
+
+Next, configure AICA Studio and add **objectdetection** under Custom Packages. After launching, you should see
+**Component Utils** under **Add Component** and be able to add *YOLO to Marker*.
+
+### Application Setup
+
+Connect the `Bounding Boxes` output of *YOLO Executor* to `JSON Input` on *YOLO to Marker* then configure it with
+**auto-configure** and **auto-activate**. You should also set the parameters based on your camera and what you are filming.
+Set `rate` to match that of *YOLO Executor*. Set the `Object to Track` parameter to "person" to track yourself in the frame. With the application running the logs should show the 3D position in the camera_frame.
+
+#### Creating a Frame and Converting it to a Signal
+
+Add in a *Hardware Interface* if it is not already there, and run the application. When it is running, select _3D Viz_ at
+the top right. Record a frame `tool0` and name it `camera_frame`, this is the position of the camera. It can be moved by
+dragging the axis markers, with the Z direction as the direction in which the camera is pointing. Note that the robot
+will move to where the objects are detected, which may be unreachable depending on the camera position, and cause the
+robot controller to stop.
+
+Stop the application, and go back to _graph_, open the yaml code and check that the recorded frame is there, for
+example:
+
+```yaml
+frames:
+  camera_frame:
+    reference_frame: world
+    position:
+      x: 0
+      y: 0.3
+      z: 0.4
+    orientation:
+      w: 0
+      x: 0
+      y: 1
+      z: 0
+```
+
+To convert the frame to a signal add *TF to Signal* from **AICA Core Components**, connect the **load** node to the *on start* block (purple square with a black play symbol), and configure the component to **auto-configure**, **auto-activate**, set `camera_frame` and `Reference frame` to world. With the application running the camera frame can be observed under
```

Comment:
```suggestion
To convert the frame to a signal add *TF to Signal* from **AICA Core Components**, connect the **load** node to the *on start* block (purple square with a black play symbol), and configure the component to **auto-configure**, **auto-activate**, set `camera_frame` and `Reference frame` to world.
```

not sure this adds a lot here, perhaps a short summary of what all this does would be enough